Source/WebKit/Platform/Cairo.cmake

```diff
@@ -0,0 +1,27 @@
+list(APPEND WebKit_SOURCES
+    Shared/API/c/cairo/WKImageCairo.cpp
+
+    UIProcess/Automation/cairo/WebAutomationSessionCairo.cpp
+)
+
+if (PORT STREQUAL "Win" OR PORT STREQUAL "PlayStation")
+    list(APPEND WebKit_SOURCES
+        UIProcess/cairo/BackingStoreCairo.cpp
+    )
+endif ()
```

Comment:
I think you should just keep this in `WC.cmake` since its required there.

Reply:
Fixed.